src/aat/resources/cucumber/questionrounds.feature

```diff
@@ -0,0 +1,67 @@
+Feature: Question round logic
+
+  Scenario: Create question round from 1 to 2
+    Given a standard online hearing is created
+    And a standard question
+    Given the question round is ' "1" '
+    When the post request is sent to create the question
+    And the response code is 200
+    Given the question round is ' "2" '
+    When the post request is sent to create the question
+    And the response code is 200
+
+  Scenario: If no jurisdiction question round limit is set then validate question round
```

Comment:
This should be rejected because QR must start from 1 and then sequentially 2, 3 ...